lectures/french_rev.md

```diff
@@ -1,1031 +1,1026 @@
+   * during times of wars, government expenditures exceed tax revenues
+      * the government finances the deficit of revenues relative to expenditures by issuing debt
+   * after a war is over, the government's tax revenues exceed its non-interest expenditures by just enough to service the debt that the government issued to finance earlier deficits
+      * thus, after a war, the government does **not** raise taxes by enough to pay off its debt
+      * instead, it just rolls over whatever debt it inherits, raising taxes by just enough to service the interest payments on that debt
+
+Eighteenth century British fiscal policy portrayed Figure {numref}`fig2` thus looks very much like a text-book example of a **tax-smoothing** model like Robert Barro's {cite}`Barro1979`.  
+
+A striking feature of the graph is what we'll nick name a  **law of gravity** for taxes and expenditures. 
+
+   * levels of government expenditures at taxes  attract each other
+   * while they can temporarily differ -- as they do during wars -- they come back together when peace returns
+
+
+
+Next we'll plot data on debt service costs as fractions of government revenues in Great Britain and France during the 18th century.
+
+```{code-cell} ipython3
+---
+mystnb:
+  figure:
+    caption: "Ratio of debt service to taxes, Britain and France"
+    name: fig1
+---
+
+# Read the data from the Excel file
+data1 = pd.read_excel('datasets/dette.xlsx', sheet_name='Debt', usecols='R:S', skiprows=5, nrows=99, header=None)
+data1a = pd.read_excel('datasets/dette.xlsx', sheet_name='Debt', usecols='P', skiprows=89, nrows=15, header=None)
+
+# Plot the data
+plt.figure()
+plt.plot(range(1690, 1789), 100 * data1.iloc[:, 1], linewidth=0.8)
+
+date = np.arange(1690, 1789)
+index = (date < 1774) & (data1.iloc[:, 0] > 0)
+plt.plot(date[index], 100 * data1[index].iloc[:, 0], '*:', color='r', linewidth=0.8)
+
+# Plot the additional data
+plt.plot(range(1774, 1789), 100 * data1a, '*:', color='orange')
+
+# Note about the data
+# The French data before 1720 don't match up with the published version
+# Set the plot properties
+plt.gca().spines['top'].set_visible(False)
+plt.gca().spines['right'].set_visible(False)
+plt.gca().set_facecolor('white')
+plt.gca().set_xlim([1688, 1788])
+plt.ylabel('% of Taxes')
+
+plt.tight_layout()
+plt.show()
+
+#plt.savefig('frfinfig1.pdf', dpi=600)
+#plt.savefig('frfinfig1.jpg', dpi=600)
+```
+
+
+Figure  {numref}`fig1` shows that interest payments on government debt (i.e., so-called ``debt service'') were high fractions of government tax revenues in both Great Britain and France.  
+
+Figure {numref}`fig2` showed us that Britain managed to balance its budget despite those large 
+interest costs. 
+<!-- #region user_expressions=[] -->
+
+
+
+
+
+But as  we'll see in our next graph, on the eve of the French Revolution in 1788, that  fiscal policy   **law of gravity** that worked so well in Britain, did not seem to be working in France.
```

Comment:
```suggestion
But as  we'll see in our next graph, on the eve of the French Revolution in 1788, that  fiscal policy   *law of gravity* that worked so well in Britain, did not seem to be working in France.
```

Reply:
I think law of gravity is a definition in this lecture given by

```
A striking feature of the graph is what we'll nick name a  **law of gravity** for taxes and expenditures. 

   * levels of government expenditures at taxes attract each other
   * while they can temporarily differ -- as they do during wars -- they come back together when peace returns
```